### PR TITLE
ketrew.3.x.x: add constraint on Lwt ≥ 3.0.0

### PR DESCRIPTION
**Pull request by smondet**
Cf. https://github.com/ocaml/opam-repository/pull/9085 (Thanks @samoht !)

### CODE REVIEW

File: packages/ketrew/ketrew.3.1.0/opam [hunk 1]
```diff
@@ -49,5 +49,6 @@ depopts: "postgresql"
 conflicts: [
   "ppx_inline_test" {>= "v0.9.0"}
   "ppx_sexp_conv" {>= "v0.9.0"}
+  "lwt" {>= "3.0.0" }
```
Comment (samoht) [inline packages/ketrew/ketrew.3.1.0/opam]: I think this one is not needed (I already added `< "3.0.0"` in the depends fields a few days ago)

File: packages/ketrew/ketrew.3.0.0/opam [hunk 2]
```diff
@@ -38,4 +38,5 @@ depopts: [
 ]
 conflicts: [
   "sexplib" {= "v0.9.0"}
+  "lwt" {>= "3.0.0" }
```
Comment (samoht) [inline packages/ketrew/ketrew.3.0.0/opam]: can you move that to the `depends` field?

File: packages/ketrew/ketrew.3.1.0/opam (hunk 1 above)
Comment (smondet) [inline packages/ketrew/ketrew.3.1.0/opam]: OK, it's still the morning here... :coffee:  :)

**Comment by smondet** [comment]
Build errors seem unrelated?


4 V1.2 Common Distro

```
[WARNING] External solver failed with inconsistent return value. Request saved
          to "/home/opam/.opam/log/solver-error-22-1.cudf"
[ERROR] External solver failure, please fix your installation and check
        /home/opam/.opam/config and variable $OPAMEXTERNALSOLVER.
        You may also retry with option --use-internal-solver
```

5 V2.0 All Distros,
 Building opam install ketrew.3.1.0 (767be7)
 
```
+- The following actions failed
| - build cmdliner     1.0.0
| - build ppx_deriving 4.1
```


5 V2.0 All Distros,
 Building opam install ketrew.3.0.0 (fd5c8e)

```
+- The following actions failed
| - build ppx_deriving 4.1

```

5 V2.0 All Distros,
 Building opam install ketrew.3.1.0 (42ca60)
```
+- The following actions failed
| - build cmdliner     1.0.0
| - build ppx_deriving 4.1
```

**Comment by samoht** [comment]
Great, thanks!

**Pull request merged**